### PR TITLE
remove unused gems from Gemfile (and indirectly loaded gems)

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

I think some of these gems should have been removed when dor_indexing gem was created.  Others should never have been added.  And others are probably vestigial.

## How was this change tested? 🤨

specs and some of the integration tests

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,8 +6,6 @@ gem 'bootsnap', '>= 1.1.0', require: false # Reduces boot times through caching;
 gem 'bunny', '~> 2.19' # for rabbitmq
 gem 'committee' # Validates HTTP requests/responses per OpenAPI specification
 gem 'config'
-gem 'dry-monads', '~> 1.3'
```
Comment (ndushay) [inline Gemfile:9]: not sure why dry-monads is explicitly mentioned here;  assume it's leftover from dor_indexing gem extraction

File: Gemfile [hunk 2]
```diff
@@ -6,8 +6,6 @@ gem 'bootsnap', '>= 1.1.0', require: false # Reduces boot times through caching;
...
-gem 'erubis'
```
Comment (ndushay) [inline Gemfile:10]: erubis has to do with erb creation, AFAICT, and isn't needed here.

File: Gemfile [hunk 3]
```diff
@@ -16,7 +14,6 @@ gem 'rack-timeout', '~> 0.5.1'
 gem 'rails', '~> 7.0.0'
 gem 'rsolr', '~> 2.0'
 gem 'sneakers', '~> 2.11'
-gem 'solrizer'
```
Comment (ndushay) [inline Gemfile:19]: again, left here from dor_indexing gem extraction

File: Gemfile [hunk 4]
```diff
@@ -29,7 +26,6 @@ group :development, :test do
   gem 'rspec_junit_formatter'
   gem 'rspec-rails', '~> 5.0'
   gem 'simplecov'
-  gem 'webmock'
```
Comment (ndushay) [inline Gemfile:32]: not used

File: Gemfile [hunk 5]
```diff
@@ -41,9 +37,7 @@ group :development do
 end
 
 group :deployment do
-  gem 'capistrano', '~> 3.0'
   gem 'capistrano-bundler'
   gem 'capistrano-passenger'
-  gem 'capistrano-shared_configs'
```
Comment (ndushay) [inline Gemfile:47]: these two are required by dlss-capistrano, so no need to explicitly require

File: Gemfile (hunk 4 above)
Comment (edsu) [reply Gemfile:32]: Is it not needed for https://github.com/sul-dlss/gis-robot-suite/blob/main/spec/spec_helper.rb#L4

**Review by edsu — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 4 above)
Comment (ndushay) [reply Gemfile:32]: @edsu this is dor-indexing-app;  the spec you mentino is in gis-robot-suite.  I'm confused.

Comment (edsu) [reply Gemfile:32]: oops!!!

**Review by edsu — APPROVED** [review]

**Pull request merged**